about.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <script
+      src="https://kit.fontawesome.com/7dcd93f003.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="about.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About Us</title>
+  </head>
+  <body>
+    <header>
+      <section class="top-nav-bar">
+        <ul class="upper-nav">
+          <li>
+            <a href=""><i class="fa fa-facebook" aria-hidden="true"></i></a>
+          </li>
+          <li>
+            <a href=""><i class="fa fa-twitter" aria-hidden="true"></i></a>
+          </li>
+          <li><a href="">Logout</a></li>
+          <li><a href="">My Page</a></li>
```

Comment:
Kindly, I suggest that when using any "A" tag, indicate the href attribute, if it doesn't contain a link, use "#".